package.json

```diff
@@ -3,6 +3,14 @@
   "homepage": "https://enei.pt",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "eslint --fix src/**/*.js",
+    "format": "prettier --write src/**/*.js"
```

Comment:
Shouldn't we also allow prettier to format .json files? If you don't want it to format `package.json` and others, we can always create a `.prettierignore`

Reply:
Prettier would crash? I tried it like this
`"format": "prettier --write './**/*.{js,json,md,jsx,scss,lolol}'"`
and got a normal output: 

> Executing task: npm run format <
>
> landing@0.1.0 format /home/miguel/dados/dev/cesium/enei2020/landing
> prettier --write './**/*.{js,json,md,jsx,scss,lolol}'
>
>CODE_OF_CONDUCT.md 62ms
>CONTRIBUTING.md 18ms
>package-lock.json 349ms
>package.json 36ms
>public/manifest.json 24ms
>README.md 38ms
>src/components/Info.js 60ms
>src/components/SocialLinks.js 30ms
>src/components/SponsorButton.js 41ms
>src/data/config.json 22ms
>src/index.js 40ms
>src/styles/_base.scss 194ms
>src/styles/_social.scss 33ms
>src/styles/app.scss 11ms
>src/util/serviceWorker.js 48ms
>
>Terminal will be reused by tasks, press any key to close it.

 (last extension is to check the problem you mention)

I also changed the path to include every file. In this case, you may want to create the aforementioned `.prettierignore` to ignore the `package*.json` files

Reply:
You are right, the problem I mention was with eslint. The most recent commit adds those types of files :+1: